cmd/check-gh-automation/main.go

```diff
@@ -39,6 +39,7 @@ type options struct {
 	releaseRepoPath string
 	flagutil.GitHubOptions
 	pluginConfig prowpluginconfig.PluginOptions
+	adminBots    flagutil.Strings
```

Comment:
I don't think we need this configuration. I would just hardcode the value of "openshift-merge-bot" as the only bot checked for branch protection.

Reply:
If you do want to keep it as a flag, name it `branch-protector-bot-name` and have it be a singular string rather than a list. We will never be doing branch protection with multiple bots.